packages/mobile/src/tokens/saga.ts

```diff
@@ -248,3 +253,23 @@ export async function getCurrencyAddress(currency: Currency) {
       return contractKit.registry.addressFor(CeloContract.StableTokenEUR)
   }
 }
+
+export function* importTokenInfo() {
+  const tokens: Token[] = yield readOnceFromFirebase('tokensInfo')
+  let token: Token
+  const kit: ContractKit = yield call(getContractKit)
+  const address: string = yield select(currentAccountSelector)
```

Comment:
small nit: The account wording here I think might cause confusion. What do you think about copying it into a `walletAddressSelector` and marking the `currentAccountSelector` as deprecated?

Reply:
this would be a big refactor, since `currentAccountSelector` is used in so many places. I'll open up a separate PR for that.

Reply:
Yeah, no I was not proposing changing everywhere, what I meant is:

Right now the selectors file says:
```
export const currentAccountSelector = (state: RootState) =>
  state.web3.account?.toLowerCase() ?? null
```
I suggested changing it to:
```
export const walletAddressSelector = (state: RootState) =>
  state.web3.account?.toLowerCase() ?? null
// @deprecated please use walletAddressSelector instead.
export const currentAccountSelector = walletAddressSelector
```

That way we don't need to make any big changes but we can start using a more descriptive selector name :)

What do you think?